app/javascript/controllers/form_date_controller.js

```diff
@@ -0,0 +1,29 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
```

Comment:
It looks like our other stimulus controllers use two-space indents and I'm seeing four-space indents here. Editor config issue? Or maybe it's time to add JS linting so we don't need to care as much? (I want to add JS linting after I get related works merged.)